lib/cisco_node_utils/interface.rb

```diff
@@ -173,6 +173,23 @@ def default_access_vlan
       config_get_default('interface', 'access_vlan')
     end
 
+    def bfd_echo
+      config_get('interface', 'bfd_echo', name: @name)
+    end
+
+    def bfd_echo=(val)
+      fail ArgumentError, 'Interface Must be physical or port-channel' unless
+        @name[/eth|port/i]
+      state = (val ? '' : 'no')
+      Feature.bfd_enable
```

Comment:
Seems like this should have a check here ( `return if state == 'no' and not bfd_echo` ) to avoid enabling bfd when the manifest asks for 'default'.